### ISSUE TITLE
TypeError: cannot use a string pattern on a bytes-like object

### ISSUE THREAD

**Issue by AngelOfTerror** [maaaaz/fgpoliciestocsv#2]
Good afternoon,
Took me some time on the Internet to find a useful script like this. But for some reason, I'm getting errors. I've tried to search on Google for this error, encountered some fixes too ... But I'm afraid I have no idea how to apply them.

```
Y:\Test>fgaddressestocsv.py -i "Y:\Test\sys_config_10.36.25.9.txt" -n
Traceback (most recent call last):
  File "Y:\Test\fgaddressestocsv.py", line 159, in <module>
    main(options, arguments)
  File "Y:\Test\fgaddressestocsv.py", line 145, in main
    results, keys = parse(options.input_file)
  File "Y:\Test\fgaddressestocsv.py", line 77, in parse
    if p_entering_address_block.search(line):

TypeError: cannot use a string pattern on a bytes-like object
```
This error happens also at other Python files in this repository.

This is the Python version I'm using;
`Python 3.5.0 (v3.5.0:374f501f4567, Sep 13 2015, 02:16:59) [MSC v.1900 32 bit (Intel)] on win32`

**Comment by maaaaz** [comment]
Hello, could you test with python 2 ?

**Comment by AngelOfTerror** [comment]
Took me some time to get them both working together, but indeed... Works fine in Python 2.

I've tried to "2to3" script, tried to convert this into Python3, but that didn't help.

For now, I'll leave the both Python versions installed. 

Thanks for the help!

PS. I've copied one of your files with 2 minor changes to also export the services. I use these csv exports for comparisons. Maybe someone else might find it useful too. Will try to share it with you.
